.circleci/config.yml

```diff
@@ -150,20 +150,42 @@ jobs:
       - store_test_results:
           path: build/reports
   
+  deploy_unstable:
+    <<: *defaults
+    description: deploy pods to trunk
+    steps: 
+      - *restore_repo
+      - restore_gems
+      - check_bundle
+      - run:
+          # without this step, the pipeline hangs during `pod repo update` after pushing (see Fastfile -> :release_pods)
+          # TODO: look into caching this 
```

Comment:
fastlane essentially just wraps CocoaPods CLI commands, so I don't think it would be a compatibility issue. 

When I was testing this in my private repo/CircleCI account, ci would get stuck typically when trying to push the second spec, `AWSPluginsCore.podspec`. One of the first steps of that would be "Cloning spec repo" and it would hang on that. After adding the step for cloning it in advance, it stopped hanging and would push all specs successfully. 

Maybe this is somehow related to my also using a private spec repo and manually specifying sources... I'm definitely happy to try without the Clone step after we get this merged in.